src/index.ts

```diff
@@ -131,7 +131,7 @@ export const main = async (): Promise<void> => {
 
           debug('groupNames', groupNames);
 
-          await Promise.all([
+          await Promise.all(
```

Comment:
this was the main problem. the array was wrapping other arrays thus not allowing Promise.all to really catch anything. took me a min to realize. 🔫